data-reconciliation-app/demo/scripts/01-data_ingest_demo.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+sudo rm -rf ./demo/Reports/*.json
+VARS_PATH="./demo/scripts/env_vars.sh"
+. "$VARS_PATH"
+
+cd "${certs}"
+
+printf "Member1 trying to ingest data \n" 
+check_eq "Member1 - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg member0) -H "Content-Type: application/json" --data-binary "@../../demo/data/member1_demo_pt1.json" $only_status_code)"
+addCheckpoint "✅ Member1 successfully ingested data"
+
+# printf "Member2 trying to ingest empty data \n"
+check_eq "Member2 - data ingest failed (data is null or empty)" "400" "$(curl $ingestUrl -X POST $(cert_arg member1) -H "Content-Type: application/json" --data-binary "" $only_status_code)"
+addCheckpoint "✅ Member2 correctly failed to ingest empty data"
+
+# printf "Member2 trying to ingest data \n" 
+check_eq "Member2 - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg member1) -H "Content-Type: application/json" --data-binary "@../../demo/data/member2_demo.json" $only_status_code)"
+printf "\n✅ Member2 successfully ingested data\n"
+
+# printf "Member3 trying to ingest data \n" 
+check_eq "Member3 - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg member2) -H "Content-Type: application/json" --data-binary "@../../demo/data/member3_demo.json" $only_status_code)"
+addCheckpoint "✅ Member3 successfully ingested data"
+
+
+curl $reportUrl -X GET $(cert_arg member0) |jq >> ../../demo/Reports/Member1_Ingest_Data_V1.json
```

Comment:
using `>>` operator will redirect the command output to a file but will append the content to the file instead of creating/replacing the current one.

I understand we are removing the files at the beginning, but if something changes in the future where we don't remove them anymore, or use the same names for files in the different steps in our flow, we may be introducing a bug with this operator

use `>` instead, and you guarantee that your report is always the one you are getting from the application.

Reply:
there are other occurrences of this operator. so I you decide for changing it, please do it for all the commands

Reply:
Changed every occurrence of >> to >